Documentation/CRDs/Cluster/external-cluster.md

```diff
@@ -163,3 +163,8 @@ spec:
   cephVersion:
     image: quay.io/ceph/ceph:v17.2.5 # Should match external cluster version
 ```
+
+##  Import K8s cluster on existing Rook-Ceph Deployment
```

Comment:
How about this subject? And we would need a paragraph describing the scenario. 
```suggestion
##  Exporting Rook to another cluster

<Insert description of the scenario>
```

Reply:
Added a para